code/modules/surgery/robotics.dm

```diff
@@ -80,6 +81,9 @@
 
 // Manipulate organs sub-surgeries
 
+/datum/surgery/intermediate/robotics/manipulate_organs
+	possible_locs = list(BODY_ZONE_PRECISE_EYES, BODY_ZONE_PRECISE_MOUTH, BODY_ZONE_CHEST,BODY_ZONE_HEAD,BODY_ZONE_PRECISE_GROIN,BODY_ZONE_L_ARM,BODY_ZONE_R_ARM)
```

Comment:
spacing

Reply:
I'm pretty sure there are a few different spots like this. I'll try to run through them and fix them up